### ISSUE TITLE
More than 5 cameras, all available for the track

### ISSUE THREAD

**Issue by Albertari** [tmeedend/camtool#17]
With 0.7.1 version you can open cameras with y,u,i,o and p keys. If you have a track with more than 5 camtool cameras, then only the first 5 camaras can be open. For example, in ddm_gts_tsukuba I have 10 cameras.

Thank you for your great work of optimization.

**Comment by tmeedend** [comment]
Hi! Thank you for your idea! Why not. With the key G,H J, K and L for example ?

**Comment by Albertari** [comment]
The ideal would be to have a button to go through sequentially all the camtool cameras that are available for the circuit.
Thank you for your replay.

**Comment by tmeedend** [comment]
We could use F10. If camtool is already activated, it activates the next data file, if not, it activates itself, like the F1 cams. What do you think?

**Comment by tmeedend** [comment]
We could use F10. If camtool is already activated, it activates the next data file, if not, it activates itself, like the F1 cams. What do you think?

**Comment by Albertari** [comment]
Yes, I think we should use F10 key for activate Camtool and if camtool is active then activate the next data file.

**Comment by Albertari** [comment]
Thank you very much for making my request.

**Comment by tmeedend** [comment]
thank you for your idea :)
